src/Tgstation.Server.Host/ServerFactory.cs

```diff
@@ -49,7 +49,10 @@ public async Task<IServer> CreateServer(string[] args, string updatePath, Cancel
 
 			var basePath = IOManager.ResolvePath();
 			IHostBuilder CreateDefaultBuilder() => Microsoft.Extensions.Hosting.Host.CreateDefaultBuilder(args)
-				.ConfigureAppConfiguration((context, configuration) => configuration.SetBasePath(basePath));
+				.ConfigureAppConfiguration((context, builder) => {
+					builder.AddYamlFile($"appsettings.{context.HostingEnvironment.EnvironmentName}.yml", optional: true, reloadOnChange: true); //Allow both appsettings.json and appsettings.yml for backwards compat
```

Comment:
```suggestion
					builder.SetBasePath(basePath);
					builder.AddYamlFile($"appsettings.{context.HostingEnvironment.EnvironmentName}.yml", optional: true, reloadOnChange: true); //Allow both appsettings.json and appsettings.yml for backwards compat
```

9001% sure this is necessary for properly locating the files in production

Reply:
It seemed to locate just fine (I started it with `dotnet Tgstation.Server.Host.dll`, not the debugger), but ill add the base yml